src/components/table/OcTable.vue

```diff
@@ -1,18 +1,35 @@
 <template>
   <table v-bind="extractTableProps()">
     <oc-thead v-if="hasHeader">
-      <oc-tr>
+      <oc-tr class="oc-table-header-row">
         <oc-th
           v-for="(field, index) in fields"
           :key="`oc-thead-${field.name}`"
           v-bind="extractThProps(field, index)"
           @click.native="$emit(constants.EVENT_THEAD_CLICKED, field)"
           @keydown.enter.native="$emit(constants.EVENT_THEAD_CLICKED, field)"
```

Comment:
Could you remove the keydown on the `th`? Right now you can `TAB` through the table header row, focussing the `<th>` first and additionally the `<oc-button>` after, so you result in two focussable elements solving sorting, while you only need one (which should be the button for the keyboard, as it's the native element for that).

Reply:
Also, please remove the `props.tabindex = "0"` from the `sort.js` mixin so that the header cell is not focusable anymore. It's still correct that the aria-sort goes to the `th`.